Tests/NetworkSessionTest.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NetworkSessionTest.swift
+//  HyperspaceTests
+//
+//  Created by Adam Brzozowski on 1/29/18.
+//  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
+//
+
+import XCTest
+import Hyperspace
+
+class NetworkSessionTest: XCTestCase {
+    
+    private let defaultRequest = URLRequest(url: NetworkRequestTestDefaults.defaultURL)
+    
+    func test_URLSessionNetworkSessionImplementation_ReturnsURLSessionDataTask() {
+
+        let networkSession: NetworkSession = URLSession.shared
+        let networkSessionDataTask: NetworkSessionDataTask = networkSession.dataTask(with: defaultRequest, completionHandler: { (data, response, error) in
+        })
```

Comment:
Not a big deal, but I think it's a little more concise to just put the closure/function suffix on the same line since there's not actually any implementation. I think you can ignore the parameters passed into the closure as well (with an `_`).
```
let networkSessionDataTask: NetworkSessionDataTask = networkSession.dataTask(with: defaultRequest, completionHandler: { _ in })
```

Reply:
Done